cfy_cluster_setup/main.py

```diff
@@ -196,11 +215,16 @@ def _install_instances(instances_dict, rpm_download_link):
 
             instance_config = '{}_config.yaml'.format(instance.name)
             dest_instance_config = join('/etc', 'cloudify', instance_config)
-            copy(join(CONFIG_FILES_DIR, instance_config), dest_instance_config)
+            instance.run_command('cp {0} {1}'.format(
+                join(CONFIG_FILES_DIR, instance_config), dest_instance_config),
+                use_sudo=True)
 
             logger.info('Installing %s', instance.name)
-            instance.run_command(
-                'cfy_manager install -c {0}'.format(dest_instance_config))
+            install_cmd = 'cfy_manager install -c {config} {verbose}'.format(
+                config=dest_instance_config,
+                verbose='-v' if verbose > 1 else '')
```

Comment:
Surely >= 1? I would've intuitively expected verbosity levels to be:
0 - not verbose
1 - somewhat verbose
2 - very verbose

Reply:
Though maybe it would be better just to make it a boolean to avoid having to juggle the documentation/etc headaches later?

Reply:
I thought about it again and yep making it boolean will make sense. I'll change it.